### PR TITLE
Stable ITM version

### PR DESCRIPTION
**Pull request by pordaniel01**
This version successfully builds and runs on ITM.

### CODE REVIEW

File: fc2k/runaway_fluid.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <paramFile>xml/runafluid.xml</paramFile>
   <defaultparamFile>xml/runafluid.xml</defaultparamFile>
   <paramXsdFile>xml/runafluid.xsd</paramXsdFile>
-  <fFile>libRunafluid.a</fFile>
+  <fFile>/gss_efgw_work/work/g2dpor/Runafluid/build/src/librunafluid.a</fFile>
```
Comment (gergopokol) [inline fc2k/runaway_fluid.xml]: Absolute path to be removed.

File: src/cpo_utils.cpp [hunk 2]
```diff
@@ -209,22 +207,22 @@ Copy data from CPO inputs to profile structure
...
+plasma_profile cpo_to_profile(const ItmNs::Itm::coreprof &coreprof, const ItmNs::Itm::coreimpur &coreimpur,
 		const ItmNs::Itm::equilibrium &equilibrium, const ItmNs::Itm::distribution &distribution){		
 
-	profile pro;
+	plasma_profile pro;
 	double number_of_parts;
 
 	// read electron density profile length of dataset: cell_length	
-	int cell_length = coreprof.ne.value.rows();
+	int plasmaLocal_length = coreprof.ne.value.rows();
```
Comment (gergopokol) [inline src/cpo_utils.cpp]: plasmaLocal_length --> plasmaProfileLength

Comment (pordaniel01) [reply src/cpo_utils.cpp]: Done

File: fc2k/runaway_fluid.xml (hunk 1 above)
Comment (pordaniel01) [reply fc2k/runaway_fluid.xml]: Done

**Review by gergopokol — APPROVED** [review]

**Pull request merged**